NewTutorials/Tutorial/PlayerViewControllerWrapper.swift

```diff
@@ -0,0 +1,57 @@
+//  Copyright © 2017 Oath. All rights reserved.
+
+import UIKit
+import OneMobileSDK
+import PlayerControls
+
+
+class PlayerViewControllerWrapper: UIViewController {
+    struct Props {
+        var isLoading = false
+        var player: Future<Result<Player>>?
+        var controlsViewController: ContentControlsViewController = DefaultControlsViewController()
+    }
+    
+    var props = Props() {
+        didSet {
+            view.setNeedsLayout()
+            
+            guard !props.isLoading else { return }
+            guard self.playerViewController?.player == nil else { return }
```

Comment:
What will happen if we remove both of these checks?

Reply:
Player will be set each time props was updated

Reply:
And I don't see a problem here.
It will not trigger player loading each time.